### ISSUE TITLE
azurerm_automation_schedule: start_time validation should just fire on create/update

### ISSUE THREAD

**Issue by khaliddermoumi** [terraform-providers/terraform-provider-azurerm#6114]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Terraform (and AzureRM Provider) Version

Terraform v0.12.23
+ provider.azurerm v2.0.0

### Description

When a schedule with "weekly" frequency is created, the only way to specify the start time on which the schedule would run (the time of the day, f. e. 05:00PM) is to set the attribute "start_time" (f. e. to "2020-03-14T05:00:00Z". Now if you create such a schedule, and the start time is in the past, running terraform creates a validation error:

Error: "start_time" is "2020-03-14T05:00:00Z" and should be at least "5m0s" in the future

This error occurs even if the resource is unchanged, creating a lock which can only be resolved if you delete the resource, or change the attribute accordingly.
The correct behaviour would be that validation only throws errors on creation, or if the value is changed.

Affected resources:
azurerm_automation_schedule

### Terraform Configuration Files

```hcl
resource azurerm_automation_schedule base_aaa_schedule_backup_volumes {
  name                    = "base-aaa-dev-schedule-backup-volumes"
  resource_group_name     = azurerm_automation_account.base_aaa.resource_group_name
  automation_account_name = azurerm_automation_account.base_aaa.name
  frequency               = "Week"
  timezone                = "UTC"
  week_days               = ["Saturday"]
  start_time              = "2020-03-14T05:00:00Z"
  expiry_time             = "2299-12-31T00:00:00Z"
  description             = "Runs each Saturday on 05:00 UTC."
}```

### Description / Feedback

When the created resource remains unchanged, no validation error should occur.

**Comment by t3mi** [comment]
@khaliddermoumi please try 2.1.0 azurerm version, it should be [fixed](https://github.com/terraform-providers/terraform-provider-azurerm/pull/5876).

**Comment by stefan-rapp** [comment]
Hi, I am still getting the same error, even with the new provider 2.5.0, like the following screenshot illustrates:

![Error_AA_Schedule](https://user-images.githubusercontent.com/45358082/79192410-e8ef2600-7e28-11ea-97d1-7175c1c38b5c.png)

Using the same code like in the post above.

```
resource "azurerm_automation_schedule" "aa_schedule" {
  name                    = "base-aaa-dev-schedule-backup-volumes"
  resource_group_name     = "rg-meggle-la-we-infra"
  automation_account_name = "aa-account-la-mon-meggle-we-infra"
  frequency               = "Week"
  timezone                = "UTC"
  week_days               = ["Saturday"]
  start_time              = "2020-03-14T05:00:00Z"
  expiry_time             = "2299-12-31T00:00:00Z"
  description             = "Runs each Saturday on 05:00 UTC."
}
```

Currently I am using the following terraform version and it does not solve the problem:
```
Terraform v0.12.24
+ provider.azurerm v2.5.0
```

Can anyone help me out? Is the bug in the 2.0.0 version of the AzureRM Provider already solved? Thx.

**Comment by stefan-rapp** [comment]
Just found the error. --> I put the wrong date in several times! :-( 
Of course it must be in the future:

![Solution_AA_Schedule](https://user-images.githubusercontent.com/45358082/79195586-cf50dd00-7e2e-11ea-9caa-4535dc42fdfd.png)

**Comment by tombuildsstuff** [comment]
Closing since this has been fixed via #5876
